### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by mfranco**
@cormacp @niallryan

### CODE REVIEW

**Comment by mfranco** [comment]
:+1:

File: documentation/source/intro.rst [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+
+Motivation
+--------------------------
+
+`Flask-Philo-Core <http://flask-philo-core.readthedocs.io/en/latest/>`_ applications
+don't include database access functionality by default because the main goal is
+to build small packages with only the required libraries for every application.
+The size of the final build is crucial for deployment in serverless environments,
+for this reason Flask-Philo-Core default environment is as mimimalistic as
```
Comment (cormacp) [inline documentation/source/intro.rst:21]: "for this reason, Flask-Philo-Core's default environment"

File: documentation/source/intro.rst [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+possible.
+
+Flask-Philo-SQLAlchemy is a Flask-Philo-Core extension that provides support
+to the `SQLAlchemy <https://docs.sqlalchemy.org>`_ library for those applications
```
Comment (cormacp) [inline documentation/source/intro.rst:25]: "that provides support for the"

File: documentation/source/intro.rst [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+that required database access.
```
Comment (cormacp) [inline documentation/source/intro.rst:26]: "that require database access"

File: documentation/source/intro.rst [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+            assert user.password == password
+            return user
+
+
+
+Views
+##############
+
+Extend ``flask_philo_sqlalchemy.http.SQLAlchemyView`` to create a class based
+view that instanciate a flask_philo_sqlalchemy.connection.ConnectionPool
```
Comment (cormacp) [inline documentation/source/intro.rst:115]: "a class based view that instanciates"

File: documentation/source/intro.rst [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+            self.sqlalchemy_pool.commit()
+            return self.json_response(status=200)
+
+
+Unit Tests
+#################
+
+
+Extend ``flask_philo_sqlalchemy.test.SQLAlchemyTestCase`` to create test cases
+that with active connections to the databases:
```
Comment (cormacp) [inline documentation/source/intro.rst:169]: "to create test cases with active connections to the databases"

**Pull request merged**